constants.py

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Loads constants for backend use."""
+
+import json
+import os
+
+class Constants(dict):
+    """Transforms dict to object, attributes can be accesed by dot notation"""
+    def __init__(self, *args, **kwargs):
+        super(Constants, self).__init__(*args, **kwargs)
+        self.__dict__ = self
```

Comment:
It should be safe on Python => 2.7.4, more info http://stackoverflow.com/a/14620633. But as I think about it I can probably replace it with:
```python
class Constants(dict): 
    __getattr__ = dict.__getitem__
```
because we only need to access the constants by dot notation and not all features of dictionary.